app/views/govuk_publishing_components/components/docs/contents_list.yml

```diff
@@ -209,6 +209,22 @@ examples:
           text: Fourth page title link
         - href: "#fifth-thing"
           text: Fifth page title link
+  with_alternate_title:
+    description: With this option, the 'Contents' title is replaced with the supplied alternate title. This should only be used when using this component as a navigation element on landing pages. Typically used in conjunction with the `alternate_active` option.
+    data:
+      alternate_title: "An alternate title"
```

Comment:
Likewise I think this could just be `title`.

Reply:
Done.